cypress/utils/utils.ts

```diff
@@ -272,11 +273,16 @@ export function applySearchFilter(
     value?: number
 ): void {
     selectFilter(filterName, identifiedRisk, value);
-    if (filterName == businessService || filterName == tag || filterName == credentialType) {
+    if (
+        filterName == businessService ||
+        filterName == tag ||
+        filterName == credentialType ||
+        filterName == artifact
+    ) {
         cy.get("div.pf-c-toolbar__group.pf-m-toggle-group.pf-m-filter-group.pf-m-show")
             .find("div.pf-c-select")
             .click();
-        if (filterName == businessService) {
+        if (filterName == businessService || filterName == artifact) {
             // ul[role=listbox] > li is for the Application Inventory page.
             // span.pf-c-check__label is for the Copy assessment page.
```

Comment:
Can we remove these commented lines ?

Reply:
@nachandr @ibragins Can we remove this commented lines, Please?

Reply:
@kpunwatk , we are using different selectors on different pages. So, the commented lines should stay . It's not an oversight. It's meant to provide information on the selectors used .

Reply:
Okay, Great..Thanks for this info @nachandr :+1: 
Then we won't remove this lines.